.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
             . venv/bin/activate
             pip install --upgrade pip
             pip install Cython
+            pip install --upgrade git+https://github.com/CEA-COSMIC/Modopt
```

Comment:
why wasn't this here before?

Reply:
So, currently, Modopt is being installed with plain pip install. 
But for my changes to take effect, we need latest Modopt codes from Master. 
Techincally we need an extra modopt version release, which I think depends on @sfarrens .
@sfarrens can we get a new Modopt release so that we have all the new operators?

Reply:
Ill keep this unresolved, if we have a modopt release before this PR, I will remove this line and merge, else we can merge this and then open a PR.

Reply:
I have removed this as we have a latest Modopt release